docs/mindsdb-docs/docs/sql/create/predictor.md

```diff
@@ -82,13 +84,15 @@ On execution, we get:
 +--------------------+----------+--------------------+--------------+---------------+-----------------+-------+-------------------+------------------+
 ```
 
-## `#!sql ... USING` Statement
+## `#!sql CREATE PREDICTOR` with the `#!sql USING` Statement
+
+### Description
 
-### `#!sql ... USING` Description
+In MindsDB, the underlying AutoML models are based on the [Lightwood](https://lightwood.io/) engine. This library generates models automatically based on the data and declarative problem definition. But the default configuration can be overridden using the `#!sql USING` statement that provides an option to configure specific parameters of the training process.
```

Comment:
In next version of MindsDB will be possible to use another ml frameworks when train predictor. I think we need indicate here that lightwood is used by default.

Reply:
Modified. Here is the corrected text:
> In MindsDB, the underlying AutoML models are based on the [Lightwood](https://lightwood.io/) engine by default. This library generates models automatically based on the data and declarative problem definition. But the default configuration can be overridden using the `#!sql USING` statement that provides an option to configure specific parameters of the training process.
> 
> In the upcoming version of MindsDB, it will be possible to choose another ML framework. Please note that the Lightwood engine is used by default.